### DISCUSSION TITLE
Creating Custom Fields

### DISCUSSION THREAD

**Discussion by pavankramadugu** [vazco/uniforms#959] [Q&A] [answered]
Hi i wanted to know on how to create custom fields from templates like prime react comps. i tried like how it is mentioned in docs. when i am trying to read props i'm getting errors. can someone link me any implementations of custom fields.

**Comment by radekmie** [comment, marked as answer]
Hi @pavankramadugu. We could say that every field is a custom one - some of them are just bundled in the "official themes". There's also a part of the tutorial that describes the process of creating a custom field [in the official docs](https://uniforms.tools/docs/tutorials-creating-custom-field). And last but not least, you can go through the [issues](https://github.com/vazco/uniforms/issues) section - there's plenty of examples there.
